internal/app/wwctl/container/rename/main.go

```diff
@@ -0,0 +1,45 @@
+package rename
+
+import (
+	"fmt"
+
+	api "github.com/hpcng/warewulf/internal/pkg/api/container"
+	"github.com/hpcng/warewulf/internal/pkg/api/routes/wwapiv1"
+	"github.com/hpcng/warewulf/internal/pkg/container"
+	"github.com/hpcng/warewulf/internal/pkg/wwlog"
+	"github.com/spf13/cobra"
+)
+
+func CobraRunE(cmd *cobra.Command, args []string) (err error) {
+	if len(args) > 2 {
+		wwlog.Warn("rename only requires 2 arguments but you provided %d arguments. Hence, they will be ignored.", len(args))
+	}
```

Comment:
This should quit out with an error if the incorrect number of arguments are provided.